public/app/plugins/panel/alertGroups/panelcfg.cue

```diff
@@ -20,7 +20,7 @@ composableKinds: PanelCfg: {
 			{
 				schemas: [
 					{
-						PanelOptions: {
+						Options: {
```

Comment:
Do we anyhow enforce/validate this? Just thinking out loud if we have any way of preventing this or `FieldConfig` naming going wilde regardless of the pattern we agreed on.